SillyBusinessInc/SillyBusinessGame
# Feat: #147 player animation

## Purpose of this PR:
this task/PR is for the player animations and it is working and good. so movement, falling, attacking, dying etc etc

## How to test:
The easiest way is to go to my scene in mustafa map the scene is called spawner.
Also check all state codes.

### links: 
[Ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/147) <-- Change the link to your ticket link
